turborepo-remote-cache/templates/storage.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.pvc.create -}}
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: turborepo-remote-cache
```

Comment:
What about using `{{ include "turborepo-remote-cache.fullname" . }}` instead of hardcoding the name? :-)

Reply:
Good callout. Using the name from the config now.